src/components/Modal/Modal.jsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+import styles from './Modal.module.css';
+
+const Backdrop = (props) => {
+    const backDropOnClick = () => {
+        props.onClose();
+    };
+    return <div className={styles['backdrop']} onClick={backDropOnClick} />;
+};
+
+const ModalOverlay = (props) => {
+    return (
+        <div className={styles['modal']}>
+            <div className={styles['content']}>{props.children}</div>
+        </div>
+    );
+};
+
+const portalOverlay = document.getElementById('overlays');
```

Comment:
Here you can create a component with bigger z index. It will be displayed above all other components